### PR TITLE
Minimum efficiency for variable refrigerant flow system

### PR DESCRIPTION
**Pull request by lymereJ**
This PR add tables of minimum efficiency requirements for variable refrigerant flow systems (VRFs) for ASHRAE Standard 90.1 and the IECC.

### CODE REVIEW

**Comment by lymereJ** [comment]
@AmmarDeh - Do you want to have a look?

File: building_energy_standards_data/database_files/hvac_minimum_requirement_variable_refrigerant_flow_systems_90_1.json [hunk 1]
```diff
@@ -0,0 +1,4429 @@
...
+        "equipment_type": "AirCooled",
+        "heating_type": "Electric Resistance or None",
+        "subcategory": "VRF multisplit system",
+        "minimum_capacity": 65000,
+        "maximum_capacity": 134999.99,
+        "start_date": "1/1/2017",
+        "end_date": "9/9/2999",
+        "minimum_seasonal_energy_efficiency_ratio": null,
+        "minimum_seasonal_energy_efficiency_ratio_2": null,
+        "minimum_energy_efficiency_ratio": 11.2,
```
Comment (AmmarDeh) [inline building_energy_standards_data/database_files/hvac_minimum_requirement_variable_refrigerant_flow_systems_90_1.json:1799]: @lymereJ Just wondering if this is similar to the AC requirements were we considered EER and IEER before but only IEER after. This applies to 90.1-2016 and 2010.

Comment (lymereJ) [reply building_energy_standards_data/database_files/hvac_minimum_requirement_variable_refrigerant_flow_systems_90_1.json:1799]: I don't think so, the language is pretty clear there, here's an example:
image

Comment (AmmarDeh) [reply building_energy_standards_data/database_files/hvac_minimum_requirement_variable_refrigerant_flow_systems_90_1.json:1799]: This is from 2016, and it is not as clear as the one you showed
image

**Pull request merged**